### DISCUSSION TITLE
SvelteKit deployments are constantly failing

### DISCUSSION THREAD

**Discussion by trm217** [vercel/community#3303] [Help] [answered]
### Summary

I am using a basic sveletkit app created with the create script within a turborepo project and I continously receive the below error when building the sveltekit app on vercel. If I redeploy it, it goes through everytime!

> No Output Directory named "public" found after the Build completed. You can configure the Output Directory in your Project Settings.

### Steps to Reproduce

Setup a turborepo app and create a sveltekit app within. Deploy it the app to vercel.

**Comment by amyegan** [comment, marked as answer]
Hey @trm217. This sounds like it could be a Turbo config issue. I recommend checking out the [Turborepo & SvelteKit Starter](https://vercel.com/templates/svelte/turborepo-sveltekit-starter) repo which deploys for me without error. 

The folks in [Turbo Discussions](https://github.com/vercel/turbo/discussions) may be able to provide more specific advice if you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) with them.
